src/Boot.java

```diff
@@ -182,7 +183,14 @@
 
     public static String
     downloadUrl(String version, String name) throws Exception {
-        return String.format("https://github.com/boot-clj/boot/releases/download/%s/%s", version, name); }
+        if (0 < version.compareTo("2.8") && name.equals("boot.jar")) {
```

Comment:
Hacky version comparison — could this be done better?
Also guarding that `boot.sh` isn't requested, it should never be as per #9 however.